### PR TITLE
feat: static template

### PR DESCRIPTION
**Pull request by danilowoz**
Closes https://github.com/codesandbox/sandpack/issues/373

### CODE REVIEW

File: sandpack-client/src/types.ts [hunk 1]
```diff
@@ -379,4 +379,5 @@ export type Template =
   | "static"
   | "solid"
   | "nextjs"
-  | "node";
+  | "node"
+  | "static";
```
Comment (coreyaus) [inline sandpack-client/src/types.ts:383]: For what it's worth, "static" is already listed as an option on line 379 so this change to the `Template` type is probably not needed 😄 Exciting to see the static template being brought into Sandpack - massively appreciate your work on this!

**Comment by hutuyishi** [comment]
When will merge this pr, man

**Comment by DeMoorJasper** [comment]
Danilo is currently on vacation so will probably be merged in 2 weeks, apologies for the bad timing on this

**Comment by dlrice** [comment]
I hope you had a good vacation Danilo. Are you back and able to merge this? Thanks!

**Review by DeMoorJasper — APPROVED** [review]

**Pull request merged**